src/App.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import { TextField } from '@material-ui/core';
+
+import './App.css'
+
+
+
+class App extends React.Component {
+    constructor(props) {
```

Comment:
This is the old way to do it - consider using class properties to make your code cleaner.
https://medium.com/front-end-weekly/class-properties-with-react-dbc9b33e901c

Reply:
done